packages/calendar/src/utils/dayTemplate.js

```diff
@@ -72,7 +72,7 @@ export function dayTemplate(day, { weekdays, monthsLabels = defaultMonthLabels }
         ?current-month=${day.currentMonth}
         ?next-month=${day.nextMonth}
       >
-        <span class="calendar__day-button__text"> ${day.date.getDate()} </span>
```

Comment:
We use this span internally for styling.

Reply:
Indeed. The sole reason for this span is a styling hook and we need it ourselves in our extension layer :)